\documentclass[oupdraft]{bio}
\usepackage[colorlinks=true,urlcolor=citecolor,linkcolor=citecolor,citecolor=citecolor]{hyperref}

% Our preamble %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newcommand{\tgt}{\textrm{tgt}}
\newcommand{\rref}{\textrm{ref}}
\newcommand{\case}{\textrm{case}}
\newcommand{\ctrl}{\textrm{ctrl}}
\newcommand{\Var}{\mathbb{V}\text{ar}}
\newcommand{\bbOne}{\mathds{1}}
\newcommand{\slfrac}[2]{\left.#1\middle/#2\right.}

\usepackage{dsfont}
% \usepackage{pdfpages}
% \usepackage{url}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


% Run analysis %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%'
%' <<test, results='hide', echo=FALSE>>=
%' getwd()
%' @
%'
%' <<run_master, echo=FALSE, results='hide', message=FALSE>>=
%' source("../scripts/master.R")
%' @
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


% Add history information for the article if required
\history{Received August 1, 2010;
revised October 1, 2010;
accepted for publication November 1, 2010}

\begin{document}

% Title of paper
\title{Variance approximation of efficiency adjusted
       $\Delta\Delta \mbox{C}_q$-values in qPCR experiments}


% List of authors, with corresponding author marked by asterisk
\author{A.E. BILGRAU$^\ast$, S. FALGREEN, A. PETERSEN, M.K. KJELDSEN,\\
J.S. B\O{}DKER,  H.E. JOHNSEN, K. DYBK\AE{}R, M. B\O{}GSTED\\[4pt]
% Author addresses
\textit{Department of Haematology, Aalborg University Hospital,
Aalborg, Denmark \\ %Sdr. Skovvej 15, 9000 Aalborg, Denmark
Department of Mathematical Sciences, Aalborg University,
Aalborg \O{}, Denmark %Fredrik Bajers Vej 7G, 9220 Aalborg \O{}, Denmark
}
\\[2pt]
% E-mail address for correspondence
{abilgrau@math.aau.dk}}


% Running headers of paper:
\markboth%
% First field is the short list of authors
{A. E. Bilgrau and others}
% Second field is the short title of the paper
{Variance approximations in qPCR experiments}

\maketitle

% Add a footnote for the corresponding author if one has been
% identified in the author list
\footnotetext{To whom correspondence should be addressed.}

\begin{abstract}
{Accurate estimation of the amplification efficiency (AE) is an important prerequisite for quantification of gene expression by quantitative polymerase chain reaction (qPCR).
The most commonly used correction strategy is to estimate the AE by dilution experiments and use it as a plug-in when efficiency correcting the   $\Delta\Delta\mbox{C}_q$-value.
In recent years it has been recommended to determine the AE with high precision as the plug-in approach do not account for its uncertainty.
In other words the method assumes an infinite precision of the AE estimate.
It has become clear, however, that determining the AE with such a high precision can require tedious laboratory work and a vast amount of biological material, which is often a scarce resource in e.g.\ molecular studies of cancer biology.
It is therefore of great importance to estimate and account for the accuracy of the AE estimate to bound the type I error and achieve the desired significance level.
We outline novel procedures to obtain variance approximations of the efficiency adjusted $\Delta\Delta\mbox{C}_q$-value based on statistical modelling in combination with either bootstrapping or the statistical $\Delta$-method.
These procedures can be utilized to obtain reliable power calculations, confidence intervals, and test statistics for the detection of differentially expressed genes by qPCR experiments.
The method is founded on a statistical model that enables the estimation of the AE and an approximations based on  of the variance of efficiency adjusted $\Delta\Delta \mbox{C}_q$-values.
We used the method to validate two recent findings, which suggested that MGST1 is differentially expressed between multiple myeloma cell lines with high and low numbers of culture initiating cells and that miRNA-127 is differentially expressed between testicular and nodal diffuse large B-cell lymphoma (DLBCL).
The presented method can be applied to evaluate the uncertainty from efficiency estimation by dilution curves and allow more reliable statistical inference based on qPCR data.}
% Keywords
{qPCR; amplification efficiency; efficiency adjusted $\Delta\Delta\mbox{C}_q$; bootstrapping, the statistical $\Delta$-method, sample size calculations.}
\end{abstract}


\section{Introduction}
% Introduction of the problem. Move from the broader issues to your specific problem.
% Craft this section carefully, setting up your argument in logical order.
% Answer the question "what is the problem and why is it important?"

Despite being an aging technique quantitative polymerase chain reaction (qPCR), ``one of the most significant biotech discoveries in our time'' \citep{Rainbow1996}, is still heavily used in molecular biology to amplify and quantify nucleic acids such as DNA or mRNA molecules.
In life sciences, qPCR is typically applied to detect and validate candidate gene transcripts that are biomarkers of diagnostic, prognostic, and even predictive value of e.g.\ infectious diseases, cancer, and genetic abnormalities.

Despite the tremendous success of qPCR, ``statistical inference considerations are still not accorded high enough priority'' \citep{Bustin2009,Bustin2010}.
This is in particular true for the estimation of the amplification efficiency.

% Relevant  Refer to relevant ideas/theories and related research by other authors

Although, efficiency calibration has been extensively treated by \cite{Pfaffl2001} or the more generalized model presented by \cite{Yuan2008},
there still seems to lack a systematic study of the unavoidable influence of uncertainty in the efficiency amplification estimate on the conclusions of qPCR experiments based on formal statistical inference.
% Statistical inference is based on the formulation of a statistical model for the phenomenon under study.
% The model includes a set of parameters to be estimated including the effects of interest.
% The inferential tools allows for achieving parameter estimates, assessing their uncertainty,  and formulating as well as testing scientific hypotheses.
In this paper we describe the estimation of the uncertainty of the efficiency amplification estimate by mixed effects models, which also allows for an appropriate handling of the technical and sample errors by either bootrapping or the statistical $\Delta$-method.

The idea of using mixed effects models for qPCR experiments is not new and has been used by e.g. \cite{fu2006} to formulate generalized estimation equations to handle within group variation.
\cite{andersen2004} and \cite{abruzzo2005} have used mixed effects modeling to identify candidate normalizing genes.
However, our idea of using bootstrapping or the statistical $\Delta$-method to handle uncertainty stemming from the efficiency estimation seems to be novel and provides an extension of \cite{Yuan2008}.

% Precise aims of the paper
The aim of the paper is to formulate a mathematical model which allows us to draw formal statistical inference about qPCR experiments with non-efficient amplifications.
This covers statistical model formulation, confidence intervals, hypothesis testing, and power calculation.
The approach is used demonstrate that multiple myeloma cancer cell lines differentially express the MGST1 gene depending on the frequency of culture initiating cells. We also used the approach  to design a study which results turned out to support the hypothesis of \cite{Robertus2009} that miRNA-127 is differentially expressed between testicular and nodal DLBCL


\section{Methods}

\subsection{The observational model}

In order to approximate the standard deviation of the efficiency adjusted
$\Delta\Delta\textrm{C}_q$-value we model the amplification process
in the following way
\begin{equation}
  \label{eq:fluorescencemodel}
  F_{C_q} = \kappa \sigma N_0 (2^{\alpha})^{C_q},
\end{equation}
where $F_{C_q}$ is the fluorescence measurement at the $C_q$'th cycle,
$\kappa$ is a proportionality constant, $N_0$ is the copy number of
interest in the
initial template, $\sigma$ is the sample specific handling error and
$\alpha$ is the percentage of the $\log_2$-AE.
In practice, the transcript abundance level is determined by the cycle $C_q$ for which given fluorescence measurement $F_{C_q}$ is reached.
We rearrange \eqref{eq:fluorescencemodel} and notice that $C_q$ can be expressed as
\[
  C_q = \frac{1}{\alpha}
        \left(\log_2 F_{C_q} - \log_2 \kappa \sigma N_0 \right).
\]
In order to estimate the relative amount of transcripts between case
and control samples for the target gene template, we assume
the amount of the reference gene template is the same in both the case
and the control, denoted by
$N_{0,\rref,\case}=N_{0,\rref,\ctrl}$,
and that the AE only vary between the target and reference gene. We then
arrive at the following expression for the $\log_2$-fold change of
the target gene template between case and controls:
\begin{align*}
  \log_2\Bigl(\frac{N_{ 0,\tgt,\case}}{N_{ 0,\tgt,\ctrl}}\Bigr)
  &= \log_2 \kappa \sigma_{ \case} N_{ 0,\tgt,\case}
    -\log_2 \kappa \sigma_{ \case} N_{ 0,\rref,\case} \\
  &\quad  - \log_2 \kappa \sigma_{\ctrl} N_{ 0,\tgt,\ctrl}
       + \log_2 \kappa \sigma_{\ctrl} N_{0,\rref,\ctrl}  \\
  &=   - \bigl\{(\alpha_{\tgt}  C_{ q,\tgt,\case}
       -  \alpha_{\rref} C_{ q,\rref,\case})
       - (\alpha_{\tgt}  C_{ q,\tgt,\ctrl}
       - \alpha_{\rref} C_{ q,\rref,\ctrl})\bigr\},
\end{align*}
assuming that the $C_q$-values have been determiend by a common flouresence level $F_{C_q}$.
This method of estimating the log relative abundance between case and controls is often referred to as the $\Delta\Delta\textrm{C}_q$-method \citep{Livak2001}, after the double difference appearing in the expression:
\begin{align}
  \label{eq:deltadelta}
  \Delta\Delta\textrm{C}_q :=
    (\alpha_{ \tgt}C_{ q,\tgt,\case} - \alpha_{ \rref}C_{ q,\rref,\case})
      - (\alpha_{ \tgt}C_{ q,\tgt,\ctrl} - \alpha_{ \rref}C_{ q,\rref,\ctrl}).
\end{align}


\subsection{Statistical model}
For ease of notation we use the abbreviation $i\in\{\tgt,\rref\}$ for gene type,
$j\in\{\case,\ctrl\}$ for sample type, $k\in\{1,\dots,K\}$ for dilution, and
$l\in\{1,\dots,L\}$ for technical replicate. In order to estimate the
$\Delta\Delta\textrm{C}_q$-value we need estimates of $\alpha_{i}$
and $C_{q,i,j}$.
A popular way of estimating $\alpha_{i}$ is by dilution curves,
i.e.\ by regressing $C_{q,i,j}$ against a series of decreasing
$N_{0,i,j}$-values, denoted by $N_{0,i,j,k}=N_{0,i,j,1}2^{-x_k}$, where
$0=x_1<\cdots<x_K<\infty$.
This can be formulated as a linear mixed effects model \citep{Pinheiro2000a},
\begin{equation*}
  C_{q,i,j,s,k} = \mu_{i,j} + A_{j/s} + \gamma_{i} x_k + \epsilon_{i,j,s,k},
\end{equation*}
where $A_{j/s}$ is a random effect from sample $s$ nested under the $j$'th sample type,
$\gamma_{i} = \alpha_{i}^{-1}$ and
$\mu_{i,j} = \alpha_{i}^{-1}(\log_2 F_{C_q} - \log_2 \kappa \sigma_{j}
N_{0,i,j})$. The random effects $A_{j/s}$'s are $N(0,\sigma_S^2)$-distributed and the
$\epsilon_{i,j,s,k}$'s are independent and $N(0,\sigma^2)$-distributed.


\subsection{Hypothesis test and confidence intervals for $\Delta\Delta\textrm{C}_q$-values by the statistical $\Delta$-method}

Let the maximum likelihood estimates of the fixed effects
\begin{equation*}
  \theta = (\mu_{\tgt,\case},  \mu_{\tgt,\ctrl},
      \gamma_{\tgt},    \mu_{\rref,\case},
            \mu_{\rref,\ctrl}, \gamma_{\rref})
\end{equation*}
be denoted by
\begin{equation*}
  \hat{\theta} = (\hat{\mu}_{\tgt,\case},  \hat{\mu}_{\tgt,\ctrl},
                  \hat{\gamma}_{\tgt},    \hat{\mu}_{\rref,\case},
                  \hat{\mu}_{\rref,\ctrl}, \hat{\gamma}_{\rref}).
\end{equation*}
Suppose that we wish to test the hypothesis
\begin{equation*}
  H_0 : c(\theta) = 0,
\end{equation*}
where $c$ is the continuously differentiable function of the
fixed effects to the $\Delta\Delta\textrm{C}_q$-value given by
\begin{equation}
  \label{eq:deltadelta2}
  c(\theta)
    =      \big\{ (\mu_{\tgt,\case}  \gamma_{ \tgt}^{-1}
               -  \mu_{\rref,\case} \gamma_{ \rref}^{-1})
               - (\mu_{\tgt,\ctrl}  \gamma_{ \tgt}^{-1}
               -  \mu_{\rref,\ctrl} \gamma_{ \rref}^{-1})\bigr\}.
\end{equation}
Then a statistic for testing $H_0$, is given by
\begin{equation*}
  t = \frac{c(\hat{\theta})}
           {\sqrt{\Var\!\big[c(\hat{\theta})\big]}}
\end{equation*}
which according to a first order Taylor series expansion of $c$ can be
approximated by
\begin{equation}
  t =
  \frac{
    c(\hat{\theta})
  }{
   \sqrt{\nabla c(\hat{\theta})^\top
        \Var[\hat{\theta}]
        \nabla c(\hat{\theta})}
  }.
  \label{eq:tstat}
\end{equation}
According to \citet[Section 2.4.2]{Pinheiro2000a}, $t$ is approximately
$t$-distributed with $\eta = S_{\case}+S_{\ctrl} -5$ degrees of freedom.
On the basis of (\ref{eq:tstat}), an approximate $(1- \alpha) 100\%$ confidence interval of $c(\theta)$ is then given by
\begin{equation*}
     c(\hat{\theta}) \pm  t_{\alpha/2,\eta}\sqrt{\nabla c(\hat{\theta})^{\top}\Var[\hat{\theta}]\nabla c(\hat{\theta})}.
\end{equation*}
Both maximum likelihood (ML) and restricted maximum likelihood estimation (REML) of mixed linear model have been implemented by the R-package \texttt{lme4} based on the \texttt{nlme} package of  \cite{Pinheiro2000a}.
Based on this package we compute the estimates $\hat{\theta}$ and $\textrm{Var}(\hat{\theta})$ and use these in the construction of test and confidence intervals for the $\Delta\Delta\textrm{C}_q$-values as described above.

\subsection{Hypothesis test and confidence intervals for $\Delta\Delta\textrm{C}_q$-values by bootstrapping}

A simple alternative to the statistical $\Delta$-method, which avoids calculating gradients and often performing better in small sample situations, is to bootstrap.

The basic idea of bootstrapping is that inference about the $\Delta\Delta\textrm{C}_q$-values can be can conducted by resampling the sample data and performing inference on the resampled data.

Nonparamtric block bootstrap samples for the case-control data are generated by sampling matched pairs of target/reference genes with replacement for cases and controls, respectively.
However, as we have only got a sinlge observation for each dilution step we chose to resample residuals from the fitted simple regression model and subsequently adding the residuals to the fitted values from the linear regression.

Hende the bootstrapped data set consists of the resampled mathced pairs and the bootstrapped standard curve. For each data set the $\Delta\Delta\textrm{C}_q$-value is calculated. For the analysis of the CIC and Testis data we draw 5000 bootstrap samples and used them to carry out statistcal inference.

First, standard errors of $\Delta\Delta\textrm{C}_q$-value is calculated by the empirical standard error of the bootstrapped samples.
Secondly, the P-value for the test for the null hypothesis of a vanishing $\Delta\Delta C_q$-value is calculated as the minimum of two times the percentile and two times one minus the percentile of 0 in the bootstrapped sample.

\subsection{Simulation of the false positive and true positive rate}

In order to properly study a statistical test procedure one needs to have an idea of the false positive rate (FPR), or specificity or type I error rate,  and the true positive rate (TPR), or  sensitivity or power.

The TPR of a statistical test is the probability that the test will correctly declare a result statistically significant given an effect size or difference of $\delta = c(\theta)$ between case and controls. Hence for qPCR-experiments the TPR is given by
\[
  \mbox{TPR} =
    P\big(t > t_{\alpha/2,\eta} \big| c(\theta) = \delta\big).
\]
A straightforward way to obtain the TPR is to simulate a number of $t$-values $t_1,\dots,t_n$ under the alternative hypothesis of $c(\theta) = \delta$ and then estimate the TPR by
\[
  \widehat{\mbox{TPR}} =
    \frac{1}{n}\sum_{i=1}^n \bbOne \!\big[ t_i >  t_{\alpha/2,\eta} \big],
\]
i.e.\ the fraction of tests correctly declared significant.

On the other hand the FPR of a statistical test is the probability that the test incorrectly will declare a result statistically significant given a vanishing effect size or difference of $ c(\theta) = 0$ between case and controls. Hence for qPCR-experiments the FPR is given by
\[
  \mbox{FPR} =
    P\big(t > t_{\alpha/2,\eta} \big| c(\theta) = 0\big).
\]
A straightforward way to obtain the FPR is to simulate a number of $t$-values $t_1,\dots,t_n$ under the null hypothesis of $c(\theta) = 0$ and then estimate the FPR by
\[
  \widehat{\mbox{FPR}} =
    \frac{1}{n}\sum_{i=1}^n \bbOne \!\big[ t_i >  t_{\alpha/2,\eta} \big],
\]
i.e.\ the fraction of tests incorrectly declared significant.

Based on the above statistical model, we computed the FPR and the TPR for a given sample sizes, sample and experimental variations, and effect sizes.

\subsection{Sample and data preparation}

\subsubsection{CIC data}

For this study 8 multiple myeloma (MM) cell lines with $>$10\% culture initiating cells (AMO-1, KMM-1, KMS-11, KMS12-PE, KMS12-BM, MOLP8, L-363, RPMI-8226), and 8 MM cell lines with $<$1\% culture initiating cells (ANBL-1, KAS-6-1, LP-1, MOLP-2, NCI-H929, OPM-2, SK-MM-2, U-266) were used. The fraction of culture initiating cells was determined by the limiting dilution method, see \cite{Lefkovits1999}.
 Total RNA was isolated from frozen cell culture pellets, using a combined method of Trizol (invitrogen) and Mirvana spin columns (Ambion). Isolated RNA was used in complementary DNA (cDNA) synthesis using SuperScript III First-Strand Synthesis Supermix (Invitrogen). As input into the total cDNA synthesis 250ng of total RNA was used. Equal amounts of random hexamers and oligo(dT) were used as primers. Quantitative real-time reverse transcriptase-polymerase chain reaction (qPCR) was performed on a Mx3000p QPCR system (Agilent Technologies/Stratgene) using the TaqMan UniversalPCR Master Mi, No AmpErase UNG, and TaqMan gene expression Assays (Applied Biosystems). The following TaqMan Gene Expression Assays were used (Assay ID numbers in parentheses): MGST1 (Hs00220393\_m1), FGFR3 (Hs00179829\_m1), WHSC1 (Hs00983716\_m1). The Two reference genes beta actin (ACTB) and GAPDH were used as endogenous controls, assay IDs  4333762-0912030 and 4333764-1111036, respectively. For each target and reference transcripts a standard curve based on seven 2-fold dilutions was constructed on a reference sample consisting of material from the AMO-1.

\subsubsection{DLBCL data}

DLBCL samples were collected from 8 testicular and 8 nodal parafin embedded lymphomas at Aalborg Hospital. Total RNA was isolated  using a combined method of  Trizol (Invitrogen) and Mirvana spin columns (Ambion). An amount of 10ng total
RNA was synthesized into first strand cDNA in a 15$\mu$L reaction using
TaqMan MicroRNA Reverse Transcription Kit (Applied Biosystems)
according to the manufactures instruction. In total 1.33$\mu$L cDNA was used as
template in the real time PCR amplification performed by Mx3000p QPCR system (Agilent Technologies/Stratgene) with sequence specific TaqMan primers (Applied Biosystems). The two microRNAs, hsa-miR-127 and  hsa-miR-143, were chosen for validation of differential expression between testicular and nodal manifestation of diffuse large B-cell lymphoma. As reference  transcripts we chose RNU-6B and RNU-24, which were less variable and equally expressed across nodal and extranodal samples among a larger list of candidate reference genes. For each target and reference transcripts a standard curve based on seven 2-fold dilutions was constructed on a reference sample consisting of a pooled material from all 16 lymphomas.

For both experiments $\rm{C}_q$ values were extracted for both the reference and target transcripts with automatic baseline and threshold selection \citep{Mx300P2013}.

\section{Results}

\subsection{CIC experiment}
The raw $C_q$-values and dilution curves are depicted in Figure \ref{fig:cqCIC} and Figure \ref{fig:powercurveCIC}, respectively. The differences in gene expressions in case genes, MGST1 and MMSNET, compared to control genes were estimated using four different methods: Naive $t$-test, regular $\Delta \Delta C_q$ estimate, and efficiency corrected estimates with and without a variance correction. These results are seen in Table \ref{tab:results}.

\subsection{DLBCL experiment}


\subsection{Simulation of FPR and TPR}

We present a simulation example of a two-sided test for the null hypothesis of a vanishing $\Delta\Delta C_q$-value at $5\%$ significance level.
The test was performed varying the number subjects divided into the case and control groups.
The effect size was set to $\delta = 10/9$, the sample and experimental variation  was set to $\sigma_{S}^2 = 1$ and $\sigma^2 = 1$, respectively.
Finally, the AE was set to 0.9. Figure \ref{fig:powercurve} shows the estimated TPR as a function of sample size.
The full-drawn lines shows the TPR for various number of dilutions steps.
The black dashed line shows the TPR if a t-test is applied directly after AE-correction. The grey dahsed line shows the TPR for the "optima situation" of perfect AE.

\section{Discussion}
% Main message
Our approach based on the statistical  $\Delta$-method to approximate the variance of efficiency adjusted $\Delta\Delta\textrm{C}_q$-values, shows that it is possible to draw statistical inference about qPCR experiments. This covers statistical model formulation, confidence intervals, hypothesis testing, and power calculation. The approach was used to validate that MGST1 is differentially expressed between multiple myeloma cell lines and to design a study which results support the hypothesis that miRNA-127 is differentially expressed between testicular and nodal DLBCL.

% Comparison with other studies
We noticed a dramatic increase in the standard error of the estimate when the
variance approximation was introduced, leading to a change of significance for the presented data set. This illustrates that it is important to consider all aspects of uncertainty when conducting efficiency correction of qPCR experiments. Problems with uncertainty in efficiency estimates should be handled by establishing well-estimated dilution curves as argued elsewhere \cite{Bustin2010}, however even in this case the presented method allows for design guidelines for power calculations and assessing the influcence of the estimated dilution curves.

% Conclusions
Despite the recent success of qPCR  much more statistical methodologcial researh is needed, in our opinion,  in order to firmly establish qPCR as a reliable quantitative method to assess the amount of nucleic acids.

\section*{Software}
All statistical analysis were done with R \citep{R2004} version 3.0.1 and the contributed lme4 package \citep{Pinheiro2000a} was applied for random effects modelling.

\section*{Supplementary Material}
An R-package with all data and scripts leading to the results of the present paper is available at \url{http://www.r-project.org}

\section*{Acknowledgments}
The research is supported by MSCNET, a translational program studying cancer stem cells in multiple myeloma supported by the EU FP6, and CHEPRE, a program studying chemosensitivity in malignant lymphoma by genomic signatures supported by The Danish Agency for Science, Technology and Innovation, as well as Karen Elise Jensen Fonden. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.\\
{\it Conflict of Interest}: None declared.


\bibliographystyle{biorefs}
\bibliography{references}

\newpage

\input{../output/Table1.tex}
\input{../output/Table2.tex}

\newpage

\begin{figure}
\begin{center}
\includegraphics[width=0.7\textwidth]{powercurve.pdf}
\end{center}
\caption{Power as function of sample size for an increasing number of dilutions in the dilution curve of a two-sided test 5\%-level test for non-vanishing $\Delta\Delta C_q$-value.}
\label{fig:powercurve}
\end{figure}

\begin{figure}
\begin{center}
\includegraphics[width=0.7\textwidth]{allgenes.pdf}
\end{center}
\caption{A: $C_q$-values: Raw $C_q$-values for each gene and sample type. B: Dilution curves: Dilution curves for ACTB, GAPDH, MGST1, and FGFR3 }
\label{fig:cqCIC}
\end{figure}

\begin{figure}
\begin{center}
\includegraphics[width=0.7\textwidth]{standard.pdf}
\end{center}
\caption{Dilution curves for ACTB, GAPDH, MGST1, and FGFR3 }
\label{fig:cqCIC2}
\end{figure}



\end{document}

