\documentclass{bio} %[oupdraft]
\usepackage[colorlinks=true,urlcolor=citecolor,linkcolor=citecolor,citecolor=citecolor]{hyperref}

% Our preamble %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\graphicspath{{../output/}}
\DeclareGraphicsExtensions{.eps}
\usepackage[outdir=./]{epstopdf}

\renewcommand{\vec}{\boldsymbol} % for bold vectors

\newcommand{\vtheta}{\vec{\theta}} % vector theta

\newcommand{\tgt}{\textrm{tgt}}
\newcommand{\rref}{\textrm{ref}}
\newcommand{\case}{\textrm{case}}
\newcommand{\ctrl}{\textrm{ctrl}}
\newcommand{\std}{\textrm{std}}
\newcommand{\Var}{\mathbb{V}\text{ar}}
\newcommand{\bbOne}{\mathds{1}}
\newcommand{\slfrac}[2]{\left.#1\middle/#2\right.}
\newcommand{\ddcq}{\Delta\Delta C_q}

\usepackage{dsfont}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


% Run analysis %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%'
%' <<run_master, echo=FALSE, results='hide', message=FALSE>>=
%' source("../scripts/master.R")  # Simply runs the master script
%' @
%'
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


% Add history information for the article if required
\history{Received August 1, 2010;
revised October 1, 2010;
accepted for publication November 1, 2010}



\begin{document}

% Title of paper
\title{Variance approximation of efficiency corrected
       $\ddcq$-values in qPCR experiments}


% List of authors, with corresponding author marked by asterisk
\author{A.E. BILGRAU$^\ast$, S. FALGREEN, A. PETERSEN, M.K. KJELDSEN,\\
J.S. B\O{}DKER,  H.E. JOHNSEN, K. DYBK\AE{}R, M. B\O{}GSTED\\[4pt]
% Author addresses
\textit{Department of Haematology, Aalborg University Hospital,
Aalborg, Denmark \\ %Sdr. Skovvej 15, 9000 Aalborg, Denmark
Department of Mathematical Sciences, Aalborg University,
Aalborg \O{}, Denmark %Fredrik Bajers Vej 7G, 9220 Aalborg \O{}, Denmark
}
\\[2pt]
% E-mail address for correspondence
{abilgrau@math.aau.dk}}


% Running headers of paper:
\markboth%
% First field is the short list of authors
{A. E. Bilgrau and others}
% Second field is the short title of the paper
{Variance approximations in qPCR experiments}

\maketitle

% Add a footnote for the corresponding author if one has been
% identified in the author list
\footnotetext{To whom correspondence should be addressed.}

\begin{abstract}
{Accurate estimation of the amplification efficiency (AE) is an important prerequisite for quantification of gene expression by quantitative polymerase chain reaction (qPCR).
The most commonly used correction strategy is to estimate the AE by dilution experiments and use it as a plug-in when efficiency correcting the $\ddcq$-value.
In recent years it has been recommended to determine the AE with high precision as the plug-in approach do not account for its uncertainty.
In other words, the method assumes an infinite precision of the AE estimate.
It has become clear, however, that determining the AE with such high precision require tedious laboratory work and vast amounts of biological material, which is often a scarce resource in e.g.\ molecular studies of cancer biology.
Violation of this assumption leads to overly optimistic estimates of the standard error of $\ddcq$, confidence intervals, and $p$-values.
Ultimately, the type I error probability is increased.
It is therefore of great importance to estimate and account for the accuracy of the AE estimate to bound the type I error rate and achieve the desired significance level.
We outline novel procedures to obtain variance approximations of the efficiency adjusted $\ddcq$-value based on statistical modelling in combination with either the statistical delta method to bootstrapping.
These procedures can be utilized to obtain more reliable power calculations, confidence intervals, and test statistics for the detection of differentially expressed genes by qPCR experiments.
The method in founded in a linear mixed effects model that enables the simultaneous estimation of the AE and $\ddcq$ and provides an approximation of the variance the efficiency corrected $\ddcq$ estimate.
The method is used to validate two recent findings which suggests that MGST1 is differentially expressed between multiple myeloma cell lines with high and low numbers of culture initiating cells and that miRNA-127 is differentially expressed between testicular and nodal diffuse large B-cell lymphoma (DLBCL).
The presented method can be applied to evaluate the uncertainty from efficiency estimation by dilution curves and allow more reliable statistical inference based on qPCR data.}
% Keywords
{qPCR; delta-delta Cq, $\ddcq$, amplification efficiency; efficiency adjusted; bootstrapping, delta method, power calculation}
\end{abstract}


\section{Introduction}
% Introduction of the problem. Move from the broader issues to your specific problem.
% Craft this section carefully, setting up your argument in logical order.
% Answer the question "what is the problem and why is it important?"

Despite being an aging technique, real-time quantitative polymerase chain reaction (qPCR)---``one of the most significant biotech discoveries in our time'' \citep{Rainbow1996}---is still heavily used in molecular biology.
qPCR is an extremely sensitive and cost-effective technique to amplify and quantify nucleic acids such as DNA or mRNA molecules.
In life sciences, qPCR is typically applied to detect candidate gene transcripts that are biomarkers of diagnostic, prognostic, and even predictive value of e.g.\ infectious diseases, cancer, and genetic abnormalities.
In recent years of the high-volume omics-data, another very important application of qPCR has arisen.
Here qPCR is used as a validation tool for the most promising results of the hypothesis generated from high-throughput screening studies such as microarrays or next generation RNA and DNA sequencing.
In precisely validation experiments the ability to control the type I error rate (i.e.\ false positive rate) is hugely important.
Validation without such an ability cannot be considered very meaningful validation and therefore conservative approaches should be taken.

Like all experiments in molecular biology and chemistry, qPCR is sensitive not only to, say, the gene transcript of interest, but also laboratory settings and experimental conditions.
There is an incredible amount sources of systematic and non-systematic variation including temperature variations, concentration differences by pipetting errors, primer affinity, and many more, all beside the genuine biological variations of interest across case and control.
Laboratory guidelines and increasingly sophisticated statistical modelling has been established to combat many of these systematic errors \citep{Bustin2010, Bustin2009}.

The regular $\ddcq$ method is very commonly as a estimate of the normalized relative expression of a target gene of interest between treated samples (case) and untreated samples (control).
The double difference is aimed for accounting for unwanted variations using one or more endogenous reference genes assumed to be approximately unchanged due to the treatment.
However, the target and reference genes might be subject to different amplification efficiencies (AE) which yield biased $\ddcq$ values.
In turn, the $\ddcq$ was modified to an AE corrected version.

Despite the tremendous success of qPCR, ``statistical inference considerations are still not accorded high enough priority'' \citep{Bustin2009, Bustin2010}.
We find this particular true for the estimation of the amplification efficiency.

Although, efficiency calibration has been extensively treated by \cite{Pfaffl2001} or the more generalized model presented by \cite{Yuan2008}, there still seems to lack a systematic study of the unavoidable influence of uncertainty in the efficiency amplification estimate on the conclusions of qPCR experiments based on formal statistical inference.
The current AE adjusted $\ddcq$ methods all do not account for the uncertainty of the estimated AE and thus effectively assumes the AE to be estimated with infinite precision.
This assumption implies a systematic underestimation of the standard error of $\ddcq$ leading to too narrow confidence intervals hereof and decreased $p$-values in inference for $\ddcq$.
If the AE is poorly determined this underestimation can drastically decrease the $p$-values and thus increase probability of type I errors.
The increased type I error probability is highly problematic when qPCR is used a validation experiments.

In this paper we describe the estimation of the uncertainty of the efficiency amplification estimate by mixed effects models, which also allows for a more appropriate handling of the technical and sample errors by either bootstrapping or the statistical delta method.

The idea of using mixed effects models for qPCR experiments is not new and has been used by e.g. \cite{fu2006} to formulate generalized estimation equations to handle within group variation.
\cite{andersen2004} and \cite{abruzzo2005} have used mixed effects modeling to identify candidate normalizing genes.
However, our idea of using bootstrapping or the statistical delta method to handle uncertainty stemming from the efficiency estimation seems to be novel and provides an extension of \cite{Yuan2008}.

% Precise aims of the paper
The aim of the paper is to formulate a mathematical model which allows us to draw formal statistical inference about qPCR experiments with non-efficient amplifications.
This covers statistical model formulation, confidence intervals, hypothesis testing, and power calculation, with special emphasis on true positive rates.
The approach is used demonstrate that multiple myeloma cancer cell lines differentially express the MGST1 gene depending on the frequency of culture initiating cells. We also used the approach  to design a study which results turned out to support the hypothesis of \cite{Robertus2009} that miRNA-127 is differentially expressed between testicular and nodal DLBCL


\section{Methods}

\subsection{The observational model}

In order to approximate the standard deviation of the efficiency adjusted
$\ddcq$-value we model the amplification process
in the following way
\begin{equation}
  \label{eq:fluorescencemodel}
  F_{C_q} = \kappa \sigma N_0 (2^{\alpha})^{C_q},
\end{equation}
where $F_{C_q}$ is the fluorescence measurement at the $C_q$'th cycle,
$\kappa$ is a proportionality constant, $N_0$ is the copy number of
interest in the
initial template, $\sigma$ is the sample specific handling error and
$\alpha$ is the percentage of the $\log_2$-AE.
In practice, the transcript abundance level is determined by the cycle $C_q$ for which given fluorescence measurement $F_{C_q}$ is reached.
We rearrange \eqref{eq:fluorescencemodel} and notice that $C_q$ can be expressed as
\begin{equation*}
  \alpha C_q = \log_2 F_{C_q} - \log_2 \kappa \sigma N_0.
\end{equation*}
In order to estimate the relative amount of target (tgt) gene transcripts between case
and control (ctrl) samples, we assume
the amount of the reference (ref) gene template is the same in both the case
and the control,
$N_{0,\rref,\case}=N_{0,\rref,\ctrl}$,
and that the AE only vary between the target and reference gene. We then
arrive at the following expression for the $\log_2$-fold change of
the target gene template between case and controls:
\begin{align*}
  \log_2\Bigl(\frac{N_{ 0,\tgt,\case}}{N_{0,\tgt,\ctrl}}\Bigr)
  &= \log_2 \kappa \sigma_{ \case} N_{0,\tgt,\case}
    -\log_2 \kappa \sigma_{ \case} N_{0,\rref,\case} \\
  &\quad
       - \log_2 \kappa \sigma_{\ctrl} N_{0,\tgt,\ctrl}
       + \log_2 \kappa \sigma_{\ctrl} N_{0,\rref,\ctrl}  \\
  &=   - \bigl\{(\alpha_{\tgt}  C_{ q,\tgt,\case}
       -  \alpha_{\rref} C_{q,\rref,\case})
       - (\alpha_{\tgt}  C_{q,\tgt,\ctrl}
       - \alpha_{\rref} C_{q,\rref,\ctrl})\bigr\},
\end{align*}
assuming that the $C_q$-values have been determined by a common florescence level $F_{C_q}$.
This method of estimating the log relative abundance between case and controls is often referred to as the $\ddcq$-method \citep{Livak2001}, after the double difference appearing in the expression:
\begin{align}
  \label{eq:ddcq}
  \ddcq :=
    (\alpha_{ \tgt}C_{ q,\tgt,\case} - \alpha_{ \rref}C_{ q,\rref,\case})
      - (\alpha_{ \tgt}C_{ q,\tgt,\ctrl} - \alpha_{ \rref}C_{ q,\rref,\ctrl}).
\end{align}
Thus we have the well-known $2^{-\ddcq}$ as the relative abundance of the original target transcript.


\subsection{Statistical model}
For ease of notation we use the abbreviations $i\in\{\tgt,\rref\}$ for gene types target and reference;
$j\in\{\case,\ctrl,\std\}$ for sample types case, control, and standard curve; $k\in\{1,\dots,K\}$ for dilution step; and $l\in\{1,\dots,L\}$ for technical replicate.
To estimate $\ddcq$, we need estimates of $\alpha_i$ and the expected $C_{q,i,j}$.
A popular way of estimating the AE $\alpha_i$ is by dilution curves,
i.e.\ by regressing $C_{q,i,j}$ against a series of decreasing
$N_{0,i,j}$-values, denoted by $N_{0,i,j,k} = N_{0,i,j,1}2^{-x_k}$, where
$0 = x_1 < \cdots < x_K < \infty$, and na\"{i}vely plugging $\hat{\alpha}_i$ into \eqref{eq:ddcq} thus disregarding its uncertainty.
The estimation of the expected $C_{q,i,j}$-values and $\alpha_i$ can however be estimated simultaneously when formulated as a linear mixed effects model (LMM) \citep{Pinheiro2000a};
\begin{equation*}
  C_{q,i,j,s,k} = \mu_{i,j} + A_{j/s} + \gamma_{i} x_k + \epsilon_{i,j,s,k},
\end{equation*}
where $A_{j/s}$ is a random effect from sample $s$ nested under the $j$'th sample type,
$\gamma_{i} = \alpha_i^{-1}$ and
$\mu_{i,j} = \alpha_i^{-1}(\log_2 F_{C_q} - \log_2 \kappa \sigma_j
N_{0,i,j})$. The random effects $A_{j/s}$ are $N(0,\sigma_S^2)$-distributed and the
error terms $\epsilon_{i,j,s,k}$ are independent and $N(0,\sigma^2)$-distributed.
LMMs provide a more correct quantification of the sources of variation and more correct evaluation of the uncertainty of $\mu_{i,j}$ and its derived quantities.

\subsection{Hypothesis test and confidence intervals for $\ddcq$-values by the statistical delta method}

Let the maximum likelihood estimates of the fixed effects
\begin{equation*}
\vtheta = (\mu_{\tgt,\case},  \mu_{\tgt,\ctrl},  \gamma_{\tgt},
           \mu_{\rref,\case}, \mu_{\rref,\ctrl}, \gamma_{\rref})^\top
\end{equation*}
be denoted by
\begin{equation*}
  \hat{\vtheta} = (\hat{\mu}_{\tgt, \case}, \hat{\mu}_{\tgt,\ctrl},  \hat{\gamma}_{\tgt},
                   \hat{\mu}_{\rref,\case}, \hat{\mu}_{\rref,\ctrl}, \hat{\gamma}_{\rref})^\top.
\end{equation*}
We wish to test the hypothesis
\begin{equation*}
  H_0 : c(\vtheta) = 0,
\end{equation*}
where $c$ is the continuously differentiable function of the
fixed effects to the $\ddcq$-value given by
\begin{equation}
  \label{eq:c}
  c(\vtheta)
    =      \big\{ (\mu_{\tgt,\case} \gamma_\tgt^{-1}
               -  \mu_{\rref,\case} \gamma_\rref^{-1})
               - (\mu_{\tgt,\ctrl}  \gamma_\tgt^{-1}
               -  \mu_{\rref,\ctrl} \gamma_\rref^{-1})\bigr\}.
\end{equation}
Then a statistic for testing $H_0$, is given by
\begin{equation*}
  t = \frac{c(\hat{\vtheta})}
           {\sqrt{\Var\!\bigl[c(\hat{\vtheta})\bigr]}}
\end{equation*}
which according to a first order Taylor series expansion of $c$ can be
approximated by
\begin{equation}
  t =
  \frac{
    c(\hat{\vtheta})
  }{
   \sqrt{\nabla c(\hat{\vtheta})^\top
        \Var[\hat{\vtheta}]
        \nabla c(\hat{\vtheta})}
  }.
  \label{eq:tstat}
\end{equation}
According to \citet[Section 2.4.2]{Pinheiro2000a}, $t$ is approximately
$t$-distributed with $\eta$ degrees of freedom.
The degrees of freedom of multilevel mixed effects models are highly non-trivial to obtain in general, however.
For the purposes of this paper, we restrict ourselves to case of a balance experimental design where $\eta$ is obtain relatively straight-forwardly.

On the basis of (\ref{eq:tstat}), an approximate $(1 - \alpha) 100\%$ confidence interval of $c(\vtheta)$ can then given by
\begin{equation*}
     c(\hat{\vtheta}) \pm
     t_{\alpha/2,\eta}\sqrt{\nabla c(\hat{\vtheta})^{\top}\Var[\hat{\vtheta}]\nabla c(\hat{\vtheta})}.
\end{equation*}
Likewise, $p$-values can be obtained by computing $P(T > |t|)$ where $T$ is $t$-distributed with $\eta$ degrees of freedom.
Both maximum likelihood (ML) and restricted maximum likelihood estimation (REML) of mixed linear model have been implemented by the R-package \texttt{lme4} based on the \texttt{nlme} package of  \cite{Pinheiro2000a}.
Based on this package we compute the estimates $\hat{\vtheta}$ and $\Var[\hat{\vtheta}]$ and use these in the construction of test and confidence intervals for the $\ddcq$-values as described above.
The needed gradient in \eqref{eq:tstat} is computed straight-forwardly from \eqref{eq:c} as
\begin{equation*}
  \nabla c(\hat{\vtheta}) =
  \left(
    \gamma_\tgt^{-1} ,
    -\gamma_\tgt^{-1} ,
    -\mu_{\tgt,\case} \gamma_\tgt^{-1} + \mu_{\tgt,\ctrl} \gamma_\rref^{-1} ,
    -\gamma_\rref^{-1} ,
    \gamma_\rref^{-1} ,
    \mu_{\rref,\case} \gamma_\tgt^{-1} - \mu_{\rref,\ctrl} \gamma_\rref^{-1}
 \right)^\top.
\end{equation*}

\subsection{Hypothesis test and confidence intervals for $\ddcq$-values by bootstrapping}

A simple alternative to the statistical delta method, which avoids calculating gradients and often performing better in small sample situations, is to bootstrap.

The basic idea of bootstrapping is that inference about the $\ddcq$-values can be can conducted by resampling the sample data and performing inference on the re-sampled data.

Non-parametric block bootstrap samples for the case-control data are generated by sampling matched pairs of target/reference genes with replacement for cases and controls, respectively.
However, as we have only got a single observation for each dilution step we chose to re-sample residuals from the fitted simple regression model and subsequently adding the residuals to the fitted values from the linear regression.

Hence the bootstrapped data set consists of the re-sampled matched pairs and the bootstrapped standard curve. For each data set the $\ddcq$-value is calculated. For the analysis of the CIC and Testis data we draw 5000 bootstrap samples and used them to carry out statistical inference.

First, standard errors of $\ddcq$-value is calculated by the empirical standard error of the bootstrapped samples.
Secondly, the $p$-value for the test for the null hypothesis of a vanishing $\ddcq$-value is calculated as the minimum of two times the percentile and two times one minus the percentile of 0 in the bootstrapped sample.


\subsection{Simulation of the false positive and true positive rate}

In order to properly study a statistical test procedure one needs to have an idea of the false positive rate (FPR), or type I error rate, as well as the true positive rate (TPR), or sensitivity or power.

In our setting, the TPR of the statistical test is the probability that the test will correctly declare a result statistically significant given an non-zero effect size $\delta = c(\theta)$ between case and controls. Hence for qPCR-experiments the TPR is given by
\begin{equation*}
  \mathrm{TPR} =
    P\bigl(t > t_{\alpha/2,\eta} \bigm| c(\theta) = \delta\bigr).
\end{equation*}
A straightforward way to obtain the TPR is to simulate a number of $t$-values $t_1,\dots,t_n$ under the alternative hypothesis of $c(\theta) = \delta$ and then estimate the TPR by
\begin{equation*}
  \widehat{\mathrm{TPR}} =
    \frac{1}{n}\sum_{i=1}^n \bbOne \!\bigl[ t_i >  t_{\alpha/2,\eta} \bigr],
\end{equation*}
i.e.\ the fraction of tests correctly declared significant.

On the other hand the FPR of a statistical test is the probability that the test incorrectly will declare a result statistically significant given a vanishing effect size or difference of $ c(\theta) = 0$ between case and controls. Hence for qPCR-experiments the FPR is given by
\begin{equation*}
  \mathrm{FPR} =
    P\bigl(t > t_{\alpha/2,\eta} \bigm| c(\theta) = 0\bigr).
\end{equation*}
A straightforward way to obtain the FPR is to simulate a number of $t$-values $t_1,\dots,t_n$ under the null hypothesis of $c(\theta) = 0$ and then estimate the FPR by
\begin{equation*}
  \widehat{\mathrm{FPR}} =
    \frac{1}{n}\sum_{i=1}^n \bbOne \!\bigl[ t_i >  t_{\alpha/2,\eta} \bigr],
\end{equation*}
i.e.\ the fraction of tests incorrectly declared significant.

Based on the above statistical model, we computed the FPR and the TPR for each discussed method under chosen some sample sizes, sample and experimental variations, and true effect sizes.



\subsection{Sample and data preparation}

\subsubsection{CIC data}

[Skriv noget om microarray-studiet]
For this study 8 multiple myeloma (MM) cell lines with $>$10\% culture initiating cells (AMO-1, KMM-1, KMS-11, KMS12-PE, KMS12-BM, MOLP8, L-363, RPMI-8226), and 8 MM cell lines with $<$1\% culture initiating cells (ANBL-1, KAS-6-1, LP-1, MOLP-2, NCI-H929, OPM-2, SK-MM-2, U-266) were used. The fraction of culture initiating cells was determined by the limiting dilution method, see \cite{Lefkovits1999}.
 Total RNA was isolated from frozen cell culture pellets, using a combined method of Trizol (invitrogen) and Mirvana spin columns (Ambion). Isolated RNA was used in complementary DNA (cDNA) synthesis using SuperScript III First-Strand Synthesis Supermix (Invitrogen). As input into the total cDNA synthesis 250ng of total RNA was used. Equal amounts of random hexamers and oligo(dT) were used as primers. Quantitative real-time reverse transcriptase-polymerase chain reaction (qPCR) was performed on a Mx3000p QPCR system (Agilent Technologies/Stratgene) using the TaqMan UniversalPCR Master Mi, No AmpErase UNG, and TaqMan gene expression Assays (Applied Biosystems). The following TaqMan Gene Expression Assays were used (Assay ID numbers in parentheses): MGST1 (Hs00220393\_m1), FGFR3 (Hs00179829\_m1), WHSC1 (Hs00983716\_m1). The two reference paraffines beta actin (ACTB) and GAPDH were used as endogenous controls, assay IDs  4333762-0912030 and 4333764-1111036, respectively. For each target and reference transcripts a standard curve based on seven 2-fold dilutions was constructed on a reference sample consisting of material from the AMO-1.

\subsubsection{DLBCL data}

DLBCL samples were collected from 8 testicular and 8 nodal paraffin embedded lymphomas at Aalborg Hospital. Total RNA was isolated  using a combined method of  Trizol (Invitrogen) and Mirvana spin columns (Ambion). An amount of 10ng total
RNA was synthesized into first strand cDNA in a 15$\mu$L reaction using
TaqMan MicroRNA Reverse Transcription Kit (Applied Biosystems)
according to the manufactures instruction. In total 1.33$\mu$L cDNA was used as
template in the real time PCR amplification performed by Mx3000p QPCR system (Agilent Technologies/Stratgene) with sequence specific TaqMan primers (Applied Biosystems). The two microRNAs, hsa-miR-127 and  hsa-miR-143, were chosen for validation of differential expression between testicular and nodal manifestation of diffuse large B-cell lymphoma. As reference  transcripts we chose RNU-6B and RNU-24, which were less variable and equally expressed across nodal and extra-nodal samples among a larger list of candidate reference genes. For each target and reference transcripts a standard curve based on seven 2-fold dilutions was constructed on a reference sample consisting of a pooled material from all 16 lymphomas.

For both experiments $\rm{C}_q$ values were extracted for both the reference and target transcripts with automatic baseline and threshold selection \citep{Mx300P2013}.

\section{Results}

\subsection{CIC experiment}
The raw $C_q$-values and dilution curves are depicted in panel A and B of Figure \ref{fig:cqCIC}, respectively.

The differences in gene expressions assessed by the $\ddcq$-values for the target genes MGST1 and MMSNET and each reference GAPDH and ACTB gene as well the combination of the reference genes, were estimated using four different methods: Paired $t$-tests and regular $\ddcq$ estimates without efficiency correction, efficiency corrected estimates with and without a variance correction by the statistical delta-method, and the mean of 5000 bootstrapped samples. These results are seen in Table \ref{table:cic}.

We notice for the paired t-test and LMM without efficiency correction that the estimates are equal, but the standard error estimate is lower for the LMM due to improved efficiency in an unbalanced case-ctrl set-up. For all combinations of reference genes the MGST1 estimate is significantly different from zero  and the MMSET estimate is not significantly different from zero.

The efficiency corrected estimates are equal for both the variance unadjusted and adjusted, but the standard deviation of the estimate and the $p$-values are higher for the adjusted values. Leading to a more conservative assessment of the difference in gene expression. The results, however, remain significant for the MGST1 gene and non-significant for the MMSET gene.

The bootstrap provides a standard deviation slightly less that the delta-method for the MGST1 gene and slightly larger for the the MMSET gene. The $p$-value is zero, probably due to the resolution provided by 500 bootstrapped samples for the MGST1 gene and slightly higher for MMSET gene.

\subsection{DLBCL experiment}

The raw $C_q$-values and dilution curves are depicted in panel A and B of Figure \ref{fig:cqTestis}, respectively.

The differences in gene expressions assessed by the $\ddcq$-values for the target genes mir127 and mir143 and each reference rnu6b and rnu24 gene as well the combination of the reference genes, were estimated using four different methods: Paired $t$-tests and regular $\ddcq$ estimates without efficiency correction, efficiency corrected estimates with and without a variance correction by the statistical delta-method, and the mean of 5000 bootstrapped samples. These results are seen in Table \ref{table:tesits}.

We notice for the paired t-test and LMM without efficiency correction that the estimates are equal, but the standard error estimate is lower for the LMM due to improved efficiency in an ubalanced case-ctrl set-up. For all combinations of reference genes the mir127 estimate is significantly different from zero  and the mir143 estimate is not significantly different from zero.

The efficiency corrected estimates are equal for both the variance unadjusted and adjusted, but the standard deviation of the estimate and the $p$-values are higher for the adjusted values. Leading to a more conservative assessment of the difference in gene expression. The results, however, remain significant for the mir127 gene and non-significant for the mir143 gene.

The bootstrap provides a standard deviation slightly higher than the delta-method for both mir127 and mir143. The $p$-value is zero for mir127 normalized with rnu6b, probably due to the resolution provided by 500 bootstrapped samples and slightly higer for the remaining scenarios.

\subsection{Simulation of FPR and TPR}

We present a simulation example of a two-sided test for the null hypothesis of a vanishing $\ddcq$-value at $5\%$ significance level.
The test was performed by 6 samples in each case and control groups and 8 dilution.
The effect size was set to $\delta = 10/9$, the sample and experimental variation  was set to $\sigma_{S}^2 = 1$ and $\sigma^2 = 0.25$, respectively.
Finally, the AE was set to 0.9.
The results are summarized in \ref{table:cic}. We notice the FPR/TPR are xx/xx, xx/xx, xx/xx, and xx/xx for the LMM, efficiency corrected without and with variance adjustment delta-method, and bootstrap method, respectively. This means the variance adjusted and bootstrapped methods controls the FPR much better than the methods without variance adjustment. This comes, however with cost of a lower true positive rate.

In Figure \ref{fig:simstudy} the performance of the FPR and TPR is shown for sample size equal to 4 and 8 in each case and control group and the number of dilution steps varying between 4 and 8.
In each panel the $p$-value cut-off is varied between 0.01, 0.05 and 0.1.


\section{Discussion}
% Main message
Our approach based on the statistical delta-method to approximate the variance of efficiency adjusted $\ddcq$-values, shows that it is possible to draw statistical inference about qPCR experiments. This covers statistical model formulation, confidence intervals, hypothesis testing, and power calculation. The approach was used to validate that MGST1 is differentially expressed between multiple myeloma cell lines and to design a study which results support the hypothesis that miRNA-127 is differentially expressed between testicular and nodal DLBCL.

% Comparison with other studies
We noticed a dramatic increase in the standard error of the estimate when the
variance approximation was introduced, leading to a change of significance for the presented data set. This illustrates that it is important to consider all aspects of uncertainty when conducting efficiency correction of qPCR experiments. Problems with uncertainty in efficiency estimates should be handled by establishing well-estimated dilution curves as argued elsewhere \cite{Bustin2010}, however even in this case the presented method allows for design guidelines for power calculations and assessing the influence of the estimated dilution curves.

% Conclusions
Despite the recent success of qPCR  much more statistical methodological research is needed, in our opinion,  in order to firmly establish qPCR as a reliable quantitative method to assess the amount of nucleic acids.

\section*{Software and supplementary Material}
All statistical analysis were done with R \citep{Rproj} version 3.1.3 and the contributed \texttt{lme4}-package \citep{Pinheiro2000a} was applied for random effects modelling.
All data, R code, LaTeX, and instructions to produce present paper and results within is freely available at \url{http://www.github.org/AEBilgrau/effadj/} or upon personal request.
The paper is generated using \texttt{knitr} (an improvement of Sweave) \citep{Xie2013, Leisch2002}.

\section*{Acknowledgments}
The research is supported by MSCNET, a translational program studying cancer stem cells in multiple myeloma supported by the EU FP6, and CHEPRE, a program studying chemo-sensitivity in malignant lymphoma by genomic signatures supported by The Danish Agency for Science, Technology and Innovation, as well as Karen Elise Jensen Fonden. The founders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.\\
{\it Conflict of Interest}: None declared.

\bibliographystyle{biorefs}
\bibliography{references}

\newpage

\input{../output/Table1.tex}
\input{../output/Table2.tex}
\input{../output/Table3.tex}

\newpage

\begin{figure}
\begin{center}
\includegraphics[width=\textwidth]{fig1}
\end{center}
\caption{
  Overview of CIC experiment data.
  A: Raw $C_q$-values for different cell lines for each gene type and sample type.
  The individual genes are not visually differentiated.
  B: Dilution data for reference genes (ACTB, GAPDH) and target genes (MGST1, MMSET).
}
\label{fig:cqCIC}
\end{figure}

\begin{figure}
\begin{center}
\includegraphics[width=\textwidth]{fig2}
\end{center}
\caption{
  Overview of testis experiment data.
  A: Raw $C_q$-values for different patient samples for each gene type and sample type.
  The individual genes are not visually differentiated here.
  B: Dilution data for reference genes (rnu24, rnu6b) and target genes (mir127, mir142).
}
\label{fig:cqTestis}
\end{figure}

\begin{figure}
\begin{center}
\includegraphics[width=\textwidth]{fig3}
\end{center}
\caption{
  Plot of the false postive rates (FPR) and true positive rates (TPR) and 95 \% confidence intervals achieved by each method at various $p$-value cut-offs (0.05, 0.01, 0.1) by simulation experiments.
  Computed from tables similar to Table \ref{tab:simexample}.
  The rates are plotted for each combination of 4 or 8 samples with 4 or 8 fold dilution curves.
}
\label{fig:simstudy}
\end{figure}



\end{document}

